transform-plugins/src/main/java/io/cdap/plugin/XMLParser.java

```diff
@@ -316,6 +344,25 @@ public Config(String inputField, String encoding, String xPathFieldMapping, Stri
       this.fieldTypeMapping = fieldTypeMapping;
       this.processOnError = processOnError;
       this.failOnArray = false;
+      this.enableExternalGeneralEntities = false;
```

Comment:
change this method to `this(inputField, encoding, ...., false, false, false, false, false)`